### DISCUSSION TITLE
Saving configuration

### DISCUSSION THREAD

**Discussion by JJ-Jackson88** [iansnyder333/ScholarlyRecommender#51] [Q&A] [answered]
How do I save my configuration so I do not always need to do it before getting my feed?

**Comment by iansnyder333** [comment, marked as answer]
The only way to currently save your configuration is to download the repository and run the application locally. The cloud app currently does not store any user data so if you refresh the page your previous actions will be lost. I am currently developing a database solution which will resolve your problem.
